src/ui/mod.rs

```diff
@@ -89,7 +89,9 @@ impl UiState {
 
         let block = Block::default().title("Files").borders(Borders::ALL);
         let list = List::new(items).block(block);
-        f.render_widget(list, chunk)
+        let mut state = ListState::default();
+        state.select(Some(self.selected));
+        f.render_stateful_widget(list, chunk, &mut state)

```

Comment:
use [`.highlight_style`](https://docs.rs/ratatui/latest/ratatui/widgets/struct.List.html#method.highlight_style) to highlight

Reply:
possibly store this instead of remaking it each time